DigitalExcellence/dex-frontend
# Feature/287 redesign filtermenu

## Description

Implemented new design of filtermenu as designed in #413 
@waltersajtos helped with changing the sorting-options from two menu's to one. 

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to project overviewpage
2. See new filtermenu
3. Check that it looks the same as the designs

## Link to issue

Closes: #287
